### PR TITLE
Feature/azure images

### PR DESCRIPTION
**Pull request by knagasekharreddy**
Please have a review and let me know for any changes.

### CODE REVIEW

File: scripts/bake-ide-ami.ps1 [hunk 1]
```diff
@@ -120,11 +120,11 @@ param (
 
 # Output the Pipeline Switch Status
 Write-Host "Pipeline Switch"
-$Pipeline | Out-Default | Write-Host | Write-Verbose
+$Pipeline | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Replace `| Write-Host` with `| Out-Default | Write-Host`

File: scripts/bake-ide-ami.ps1 [hunk 2]
```diff
@@ -187,7 +187,7 @@ try
     if ( $UploadInstallationImageChanges -and !$InstallScalable ) {
         Write-Host ("$(Log-Date) Upload any changes to current installation image")
 
-        Write-Verbose ("Test if source of DVD image exists") | Out-Default | Write-Host
+        Write-Host ("Test if source of DVD image exists") | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Pipes are not required for this case

File: scripts/bake-ide-ami.ps1 [hunk 3]
```diff
@@ -233,7 +233,7 @@ try
     # First image found is presumed to be the latest image.
     # Force it into a list so that if one image is returned the variable may be used identically.
 
-    Write-Verbose ("Locate image Name $AmazonAMIName") | Out-Default | Write-Host
+    Write-Host ("Locate image Name $AmazonAMIName") | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: The Pipes are not required

File: scripts/bake-ide-ami.ps1 [hunk 4]
```diff
@@ -272,7 +272,7 @@ try
         $VmResourceGroup = "BakingDP-$VersionText"
 
         # Create or update the resource group using the specified parameter
-        New-AzResourceGroup -Name $VmResourceGroup -Location $Location -Verbose -Force -ErrorAction Stop | Out-Default | Write-Host | Write-Verbose
+        New-AzResourceGroup -Name $VmResourceGroup -Location $Location -Verbose -Force -ErrorAction Stop | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Replace `| Write-Host` with `| Out-Default | Write-Host`

File: scripts/install-lansa-base.ps1 [hunk 5]
```diff
@@ -223,7 +223,7 @@ try
 
         try {
             Start-Sleep -Seconds 20
-            "$(Log-Date) Add a 20s sleep before installing kdiff3 from choco" | Out-Default | Write-Host | Write-Verbose
+            "$(Log-Date) Add a 20s sleep before installing kdiff3 from choco" | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/install-lansa-base.ps1]: Replace `| Write-Host` with `| Out-Default | Write-Host`

File: scripts/get-azure-sas-token.ps1 [hunk 6]
```diff
@@ -56,7 +56,7 @@ try {
     $token = New-AzStorageContainerSASToken -Context $storageContext -Name $ContainerName -Permission rl -ExpiryTime $endTime -StartTime $startTime
 
     # Pipeline uses the ImageUrl variable with value $uri$token
-    Write-Output "##vso[task.setvariable variable=ImageUrl;isOutput=true]'$uri$token'" | Out-Default | Write-Host
+    Write-Host "##vso[task.setvariable variable=ImageUrl;isOutput=true]'$uri$token'" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/get-azure-sas-token.ps1]: Remove Pipes

File: scripts/dot-CommonTools.ps1 [hunk 7]
```diff
@@ -224,10 +224,10 @@ param (
     )
 
     # OK and Cancel buttons
-    Write-Host "$(Log-Date) $Message" | Out-Default | Write-Verbose
+    Write-Host "$(Log-Date) $Message" | Out-Default | Write-Host
 
     if ($Pipeline) {
-        Write-Host "$(Log-Date) Skipped the MessageBox for Pipeline" | Out-Default | Write-Verbose
+        Write-Host "$(Log-Date) Skipped the MessageBox for Pipeline" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/dot-CommonTools.ps1]: Remove Pipes

File: scripts/dot-CommonTools.ps1 [hunk 8]
```diff
@@ -224,10 +224,10 @@ param (
     )
 
     # OK and Cancel buttons
-    Write-Host "$(Log-Date) $Message" | Out-Default | Write-Verbose
+    Write-Host "$(Log-Date) $Message" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/dot-CommonTools.ps1]: Remove Pipes

File: scripts/dot-CommonTools.ps1 [hunk 9]
```diff
@@ -160,7 +160,7 @@ function Connect-RemoteSession
         Sleep -Seconds 10
     }
 
-    Write-Host "$(Log-Date) $Script:publicDNS remote PS connection obtained" | Out-Default | Write-Verbose
+    Write-Host "$(Log-Date) $Script:publicDNS remote PS connection obtained" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/dot-CommonTools.ps1]: Remove Pipes

File: scripts/dot-CommonTools.ps1 [hunk 10]
```diff
@@ -150,7 +150,7 @@ function Connect-RemoteSession
     # Wait until PSSession is available
     while ($true)
     {
-        "$(Log-Date) Waiting for remote PS connection" | Out-Default | Write-Host | Write-Verbose
+        "$(Log-Date) Waiting for remote PS connection" | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/dot-CommonTools.ps1]: Replace `| Write-Host` with `| Out-Default | Write-Host`

File: scripts/bake-scalable-azure-image-pipeline.ps1 [hunk 11]
```diff
@@ -79,4 +79,4 @@ Bake-IdeMsi -VersionText $VersionText `
             -OnlySaveImage $false `
             -CreateVM $true `
             -Pipeline:$true `
-            -AtomicBuild:$AtomicBuild | Out-Default | Write-Host | Write-Verbose
+            -AtomicBuild:$AtomicBuild | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-scalable-azure-image-pipeline.ps1]: Replace `| Write-Host` with `| Out-Default | Write-Host`

File: scripts/bake-ide-ami.ps1 [hunk 12]
```diff
@@ -833,7 +833,7 @@ $jsonObject = @"
 
         Write-Host "$(Log-Date) Starting Azure Image Creation"
 
-        Write-Verbose "$(Log-Date) Delete image if it already exists" | Out-Default | Write-Host
+        Write-Host "$(Log-Date) Delete image if it already exists" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Remove the Pipes

File: scripts/bake-ide-ami.ps1 [hunk 13]
```diff
@@ -791,7 +791,7 @@ $jsonObject = @"
         ReConnect-Session
 
         Write-Host "$(Log-Date) Sysprep"
-        Write-Verbose "Use Invoke-Command as the Sysprep will terminate the instance and thus Execute-RemoteBlock will return a fatal error" | Out-Default | Write-Host
+        Write-Host "Use Invoke-Command as the Sysprep will terminate the instance and thus Execute-RemoteBlock will return a fatal error" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Remove the Pipes

File: scripts/bake-ide-ami.ps1 [hunk 14]
```diff
@@ -750,7 +750,7 @@ $jsonObject = @"
 
         if ( $InstallIDE -or ($InstallSQLServer -eq -$false) ) {
             Invoke-Command -Session $Script:session {
-                Write-Verbose "$(Log-Date) Switch Internet download security warning back on" | Out-Default | Write-Host
+                Write-Host "$(Log-Date) Switch Internet download security warning back on" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Remove the Pipes

File: scripts/bake-ide-ami.ps1 [hunk 15]
```diff
@@ -635,8 +635,8 @@ $jsonObject = @"
         # No harm installing this again if its already installed
         if ( $InstallIDE -eq $true) {
             if ( $Win2012 ) {
-                Write-Verbose "$(Log-Date) Run choco install jdk8 -y. No idea why it fails to run remotely!" | Out-Default | Write-Host
-                Write-Verbose "$(Log-Date) Maybe due to jre8 404? Give it a go when next build IDE" | Out-Default | Write-Host
+                Write-Host "$(Log-Date) Run choco install jdk8 -y. No idea why it fails to run remotely!" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Remove the Pipes

File: scripts/bake-ide-ami.ps1 [hunk 16]
```diff
@@ -568,7 +568,7 @@ $jsonObject = @"
             Execute-RemoteInitPostGit
 
             Execute-RemoteBlock $Script:session {
-                Write-Verbose "$(Log-Date) Refreshing git tools repo" | Out-Default | Write-Host
+                Write-Host "$(Log-Date) Refreshing git tools repo" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Remove the Pipes

File: scripts/bake-ide-ami.ps1 [hunk 17]
```diff
@@ -482,11 +482,11 @@ $jsonObject = @"
                 New-ItemProperty -Path $lansaKey  -Name 'InstallSQLServer' -PropertyType DWord -Value $using:InstallSQLServer -Force | Out-Default | Write-Host
                 New-ItemProperty -Path $lansaKey  -Name 'Platform' -PropertyType String -Value $using:Platform -Force | Out-Default | Write-Host
 
-                Write-Verbose "Switch off Internet download security warning" | Out-Default | Write-Host
+                Write-Host "Switch off Internet download security warning" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Remove pipes

File: scripts/bake-ide-ami.ps1 [hunk 18]
```diff
@@ -465,7 +465,7 @@ $jsonObject = @"
 
         Execute-RemoteBlock $Script:session {
             try {
-                Write-Verbose ("Save S3 DVD image url and other global variables in registry") | Out-Default | Write-Host
+                Write-Host ("Save S3 DVD image url and other global variables in registry") | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Write Pipes

File: scripts/bake-ide-ami.ps1 [hunk 19]
```diff
@@ -311,14 +311,14 @@ try
...
-        Write-Verbose "$(Log-Date) Create VM" | Out-Default | Write-Host
+        Write-Host "$(Log-Date) Create VM" | Out-Default | Write-Host
         $SecurePassword = ConvertTo-SecureString $Script:password -AsPlainText -Force
         $Credential = New-Object System.Management.Automation.PSCredential ($AdminUserName, $SecurePassword);
 
         $NicName = "bakingNic-$($Script:vmname)"
         $nic = Get-AzNetworkInterface -Name $NicName -ResourceGroupName $VmResourceGroup -ErrorAction SilentlyContinue
         if ( $null -eq $nic ) {
-            Write-Verbose "$(Log-Date) Create NIC" | Out-Default | Write-Host
+            Write-Host "$(Log-Date) Create NIC" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Remove the Pipes

File: scripts/bake-ide-ami.ps1 [hunk 20]
```diff
@@ -311,14 +311,14 @@ try
             }
         }
 
-        Write-Verbose "$(Log-Date) Create VM" | Out-Default | Write-Host
+        Write-Host "$(Log-Date) Create VM" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Remove the Pipes

File: scripts/bake-ide-ami.ps1 [hunk 21]
```diff
@@ -284,7 +284,7 @@ try
             
             # Create or update the storage account using the specified parameter
             $templateUri = "$(Split-Path -Parent $script:IncludeDir)\ARM\storage-account\stagingdp.json"
-            New-AzResourceGroupDeployment -ResourceGroupName $StorageAccountResourceGroup -TemplateFile $templateUri -TemplateParameterObject @{name = $StorageAccountName} | Out-Default | Write-Host | Write-Verbose
+            New-AzResourceGroupDeployment -ResourceGroupName $StorageAccountResourceGroup -TemplateFile $templateUri -TemplateParameterObject @{name = $StorageAccountName} | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Replace `| Write-Host` with `| Out-Default | Write-Host`

File: scripts/bake-ide-ami.ps1 [hunk 22]
```diff
@@ -294,7 +294,7 @@ try
         $publicDNSName = "bakingpublicdnsDP-$($Script:vmname)"
 
         if ( $CreateVM -and -not $OnlySaveImage) {
-            Write-Verbose "$(Log-Date) Delete VM if it already exists" | Out-Default | Write-Host
+            Write-Host "$(Log-Date) Delete VM if it already exists" | Out-Default | Write-Host
```
Comment (VarunChopraAccolite) [inline scripts/bake-ide-ami.ps1]: Remove the Pipes

File: Tests/ImageVm.Tests.ps1 [hunk 23]
```diff
@@ -28,7 +28,7 @@ Describe "VM Tests" {
             # Log-Date can't be used yet as Framework has not been loaded
 
             Write-Host "Initialising environment - presumed not running through RemotePS"
-            Write-Host $PSCommandPath | out-default | write-verbose
+            Write-Host $PSCommandPath | out-default | write-Host
```
Comment (VarunChopraAccolite) [inline Tests/ImageVm.Tests.ps1]: Remove the Pipes

**Comment by knagasekharreddy** [comment]
Updated the changes please have a review and let me know for the further changes.

**Comment by knagasekharreddy** [comment]
Updated the changes please have a review and let me know if any thing required.

**Review by VarunChopraAccolite — APPROVED** [review]

**Pull request merged**